developers/guides/explainers/wallets-explained.md

```diff
@@ -0,0 +1,226 @@
+---
+date: Jun 15, 2022
+difficulty: intro
+title: "Wallets Explained"
+description:
+  "Tell you a secret, wallets are misleading. They do not store your crypto. In fact, they do not hold anything. So why do we need even them in dapps and blockchain?"
+tags:
+  - wallets
+keywords:
+  - intro
+  - blockchain
+  - wallets
+  - blockchain explainers
+---
+
+![](https://solana.ghost.io/content/images/2022/06/Wallets---3-Blockchain-Puzzle.png)
+
+In this article, I will take you out of the rabbit hole. We will, together, put back the pieces of all the blockchain concepts by uncovering the true purpose of wallets.
+
+Hint: Wallets are not here to show your losses in a bear market 😅, they are here to access the blockchain and build!
+
+💰 Puzzle Piece 1: The Balance Myth
+===================================
+
+🤫 Tell you a secret, wallets are misleading. They do not store your crypto. In fact, they do not hold anything. So why do we need even them in dapps and blockchain?
+
+![](https://solana.ghost.io/content/images/2022/06/Wallets---1-Phantom-Balance.png)
+
+Skeptical? Take a look at this: [https://explorer.solana.com/address/E35325pbtxCRsA4uVoC3cyBDZy8BMpmxvsvGcHNUa18k?cluster=devnet](https://explorer.solana.com/address/E35325pbtxCRsA4uVoC3cyBDZy8BMpmxvsvGcHNUa18k?cluster=devnet&ref=solana.ghost.io). If my crypto were stored inside a wallet, how come you can just look up what's inside my wallet on the Internet?
```

Comment:
Can you remove all the `ref=solana.ghost.io` tags from each of the links in this doc?